{{cookiecutter.github_project_name}}/README.md

```diff
@@ -9,17 +9,16 @@
 
 ## Installation
 
-A typical installation requires the following three commands to be run:
+A typical installation requires the following commands to be run:
 
 ```bash
 pip install {{ cookiecutter.python_package_name  }}
-jupyter nbextension install --py [--sys-prefix|--user|--system] {{ cookiecutter.python_package_name  }}
 jupyter nbextension enable --py [--sys-prefix|--user|--system] {{ cookiecutter.python_package_name  }}
 ```
 
 Or, if you use jupyterlab:
 
 ```bash
 pip install {{ cookiecutter.python_package_name  }}
-jupyter labextension install {{ cookiecutter.jlab_extension_name }}
+jupyter labextension install @jupyter-widgets/jupyterlab-manager
```

Comment:
Consider it a feature request to have installation of a labextensions automatically install labextension dependencies as well :) If those need to be declared separately in the package.json, I'm OK with that.